docs/source/ci-cd.mdx

```diff
@@ -85,13 +135,10 @@ echo "$HOME/.rover/bin" >> $GITHUB_PATH
 
 Because the `rover config auth` command is interactive, you need to [authenticate using an environment variable](./configuring#with-an-environment-variable) in your project settings.
 
-GitHub actions uses [project environments](https://docs.github.com/en/actions/reference/environments) to set up secret environment variables. In your action, you choose a `build.environment` by name and set `build.env` variables using the saved secrets.
+GitHub Actions uses [project environments](https://docs.github.com/en/actions/reference/environments) to set up secret environment variables. In your action, you choose a `build.environment` by name and set `build.env` variables using the saved secrets.
 
 </Note>
 
-The following is a full example script, showing how to choose an `apollo` environment and set an `APOLLO_KEY` variable:
```

Comment:
Curious about the removal of this description of the example below?

Reply:
It feels superfluous, but if you think it's helpful, happy to re-add!

Reply:
Fine as is.